message/src/main/java/DummyClass.java

```diff
@@ -0,0 +1,5 @@
+public class DummyClass {
```

Comment:
could you please rename the class to be more descriptive why we are needing it and what it is going to be used for